src/main/kotlin/graphics/scenery/volumes/TransferFunctionEditor.kt

```diff
@@ -166,10 +123,10 @@ class TransferFunctionEditor constructor(
             override fun mouseReleased(e: MouseEvent) {
                 mouseTargetCP.itemIndex = -1
             }
-            override fun mousePressed(e: MouseEvent) {}
-            override fun mouseClicked(e: MouseEvent) {}
-            override fun mouseEntered(e: MouseEvent) {}
-            override fun mouseExited(e: MouseEvent) {}
+            override fun mousePressed(e: MouseEvent) {/*noop*/}
```

Comment:
These guys should have docs on what they're (not) doing

Reply:
like: "if a mouse button is pressed nothing happens because we dont need this functionality at the moment"? really?